app/models/organization.rb

```diff
@@ -41,6 +41,7 @@ class Organization < ApplicationRecord
   after_save  :bust_cache
   before_save :generate_secret
   before_validation :downcase_slug
```

Comment:
Will this callback case `slug_changed?` to always be true?

Reply:
Oh, good point.

Reply:
A quick test in console says this shouldn't affect it:

![image 2018-12-28 at 9 58 01 am](https://user-images.githubusercontent.com/17884966/50519078-23dc1780-0a87-11e9-8f2b-6a523eb65b04.png)

Reply:
👍